testnet/stacks-node/src/tests/signer.rs

```diff
@@ -156,9 +157,15 @@ fn test_stackerdb_dkg() {
     if env::var("BITCOIND_TEST") != Ok("1".into()) {
         return;
     }
+
+    tracing_subscriber::registry()
+        .with(fmt::layer())
+        .with(EnvFilter::from_default_env())
+        .init();
+
     // Generate Signer Data
     let num_signers: u32 = 16;
-    let num_keys: u32 = 4000;
```

Comment:
We don't want to keep the 4000 keys?

Reply:
I lowered the number so the test would go faster, but happy to change it back.  It takes ~60 seconds to bootstrap the node, so I guess it doesn't matter if it takes another 290 to finish?